ergotree-ir/src/address.rs

```diff
@@ -134,6 +240,46 @@ impl Address {
                 .unwrap())
             }
             Address::P2S(bytes) => ErgoTree::sigma_parse_bytes(bytes),
+            Address::P2SH(script_hash) => {
+                let get_var_expr = Expr::GetVar(GetVar {
+                    var_id: 1,
+                    var_tpe: SType::SColl(Box::new(SType::SByte)),
+                });
+                let hash_expr = Expr::CalcBlake2b256(CalcBlake2b256 {
+                    input: Box::new(get_var_expr),
+                });
+                let slice_expr = Expr::Slice(Slice {
+                    input: Box::new(hash_expr),
+                    from: Box::new(Expr::Const(Constant {
```

Comment:
```suggestion
                    from: Box::new(Expr::Const(0i32.into()))
```
should work

Reply:
Even `Box::new(0i32.into())` should work. If not, it could be fixed with `impl<T: Into<Constant>> Into<Expr> for T {...}`, feel free to add it.

Reply:
Fixed, along with the `Slice.until` 24i32.into() expression.